framework/tests/all-systems/GetClassTest.java

```diff
@@ -6,7 +6,9 @@ void context() {
         Integer i = 4;
         i.getClass();
         Class<?> a = i.getClass();
+        @SuppressWarnings("fenum:assignment.type.incompatible")
```

Comment:
A brief comment for why the supressions are ok would be useful. Or alternatively, add the `@FenumTop` to the upper bound?